index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Benson Portfolio</title>
+</head>
+<body>
+    <header id="header">
+        <section id="headertoolbar">
```

Comment:
Kindly remove the `section` tag inside the `header` tag to follow HTML best practices, note by following that your project will look more professional. 😄

Reply:
> Kindly remove the `section` tag inside the `header` tag to follow HTML best practices, note by following that your project will look more professional. 😄

- Replaced with `div` tag with a descriptive class attribute.